### ISSUE TITLE
multi-line text , createStdout   Can not display  /r/n

### ISSUE THREAD

**Issue by lutaooooo** [ndabAP/vue-command#209]

**Comment by ndabAP** [comment]
Could you please specify more what's the issue.

**Comment by ndabAP** [comment]
If I understood you correctly try this:

```js
createStdout(`first line
next line`)
```

**Comment by JoshuaBThompson** [comment]
@ndabAP I'm having the same issue. 
When I push: 
```js
this.history.push(createStdout(`first line
 next line`))
```

what I see on the terminal is:
```bash
>> 
first line next line
>>
```

**Comment by ndabAP** [comment]
You should be able to use plain HTML like here:

https://github.com/ndabAP/vue-command/blob/master/src/hosted/App.vue#L55

**Comment by JoshuaBThompson** [comment]
@ndabAP oh yeah I see that createStdout useds innerHtml to insert text so that’s it. If you use innerText instead that should allow for newline and carriage return to be interpreted as expected right?

**Comment by ndabAP** [comment]
@JoshuaBThompson it sounds great but I think it should be optional. Would you make a PR for this?

**Comment by JoshuaBThompson** [comment]
@ndabAP yeah, totally agree that it should be optional. Ok, I'll make a pull request soon.

**Comment by JoshuaBThompson** [comment]
@ndabAP pull request ready :)
https://github.com/ndabAP/vue-command/pull/211

**Comment by JoshuaBThompson** [comment]
@ndabAP updated the code a bit in the pull request. Removed the innerText option from createStderr. Only createStdout uses this now. See PR:
https://github.com/ndabAP/vue-command/pull/212
